tests/TESTINS5.MLC

```diff
@@ -481,9 +490,13 @@ BIC_OK2  EQU    *
          RT1   CC2
          CLG   R2,=FL8'6'
          RT1    CC0 
+         ESPIE SET,ESPIE_EXIT,8,PARAM=MSGRKC_CC3
          LMG   R5,R6,=XL8'7FFFFFFFFFFFFFFF,2'
          SR       R0,R0
          MSGRKC  R2,R5,R6
+         RT1    ABORT
+MSGRKC_CC3 EQU *
+          WTO  'MSGRKC_CC3 OVERFLOW TEST OK'
```

Comment:
I worry that the WTO changed the CC, but apparently it is still 3 after the WTO.